README.md

```diff
@@ -193,6 +193,7 @@ constructorio.tracker.trackConversion('dogs', {
 | --- | --- |
 | `name` | string |
 | `customer_id` | string |
+| `variation_id` | string | ✔ |
```

Comment:
Even with the checkmark there, it still displays as if it were only two columns. So it should work fine when everything is merged into master 👍

Reply:
Sweet - let me get that merged and then it should make things easier for you over here.